corehq/apps/data_cleaning/templates/data_cleaning/forms/manage_filters_form.html

```diff
@@ -15,7 +15,10 @@
         hq-hx-action="reorder_filters"
         hx-trigger="end"
       >
-        <div class="list-group">
+        <div
+          class="list-group"
+          x-htmx-sortable=""
```

Comment:
Nitpick: What would you think of passing in an empty config object? The totally blank attribute is a little odd.

Reply:
you can just do `x-htmx-sortable` with no `=""` but it's technically not valid HTML. I don't think it is odd